src/test/java/core/basesyntax/service/impl/TransactionProcessorImplTest.java

```diff
@@ -0,0 +1,70 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.Transaction;
+import core.basesyntax.service.TransactionProcessor;
+import core.basesyntax.service.strategy.OperationStrategy;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class TransactionProcessorImplTest {
+    private static List<Transaction> validTransactions;
+    private static Map<Transaction.Operation, OperationStrategy> handlers;
+    private static List<String> resultList;
+    private static TransactionProcessor transactionProcessor;
+
+    @BeforeAll
+    static void beforeAll() {
+        resultList = new ArrayList<>();
+        OperationStrategy operationStrategy = (fruit, quantity) -> resultList.add(fruit);
+        handlers = Map.of(
+                Transaction.Operation.BALANCE, operationStrategy,
```

Comment:
You don't need to test logic of handlers of strategy here.

![image](https://github.com/mate-academy/jv-fruit-shop-tests/assets/82539553/4e1d4863-c055-426e-8198-cd40eed00c91)